org.schabi.NewPipe.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>org.schabi.NewPipe</id>
```

Comment:
`schabi.org` doesn't seem to exist.

> The domain used for the application ID is [controlled by the application developers](https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository) and the [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

This is not a suggestion.

Reply:
https://newpipe.schabi.org/ exists, but it just redirects to https://newpipe.net/

Reply:
`schabi.org` doesn't